pkg/proxy/winkernel/proxier_test.go

```diff
@@ -106,11 +105,9 @@ func NewFakeProxier(syncPeriod time.Duration, minSyncPeriod time.Duration, clust
 		networkType: networkType,
 	}
 	proxier := &Proxier{
-		portsMap:            make(map[localPort]closeable),
-		serviceMap:          make(proxyServiceMap),
-		serviceChanges:      newServiceChangeMap(),
-		endpointsMap:        make(proxyEndpointsMap),
-		endpointsChanges:    newEndpointsChangeMap(hostname),
+		portsMap:            make(map[utilproxy.LocalPort]utilproxy.Closeable),
+		serviceMap:          make(proxy.ServiceMap),
+		endpointsMap:        make(proxy.EndpointsMap),
```

Comment:
I think it would be good to add some tests to cover EndpointSlice functionality. This is not a pretty example, but with the iptables proxier we have a test covering EndpointSlice usage all the way through to final iptables config that is computed: https://github.com/kubernetes/kubernetes/blob/master/pkg/proxy/iptables/proxier_test.go#L2412-L2504